CHANGES.md

```diff
@@ -6,6 +6,10 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+* Updated telegraf to process mesos operations metrics (DCOS_OSS-5023)
+
+* Upgraded CockroachDB to version [2.1.7](https://www.cockroachlabs.com/docs/releases/v2.1.7.html). (DCOS_OSS-5262)
```

Comment:
Oh, that line got in here by accident, right?

Reply:
Let me add a commit and remove it.

Reply:
does this line still need to be removed?

Reply:
I rebased and it came back :sleepy:

Reply:
OK, it's gone for real now.